### ISSUE TITLE
OBS goes into broken state after trying to stop recording via WebSocket

### ISSUE THREAD

**Issue by mbakumenkov** [obsproject/obs-studio#9635]
### Operating System Info

Windows 10

### OBS Studio Version

29.1.3

### OBS Studio Log URL

https://obsproject.com/logs/gpHOVHWAfiMIhVa5

### Expected Behavior

Recording stops correctly, the program continues to work normally

### Current Behavior

Recording control button goes to "Stopping recording...", also, the stream indicator (the square in the lower right corner) lights up green, but the bitrate indicator shows 0 bit/s, so in fact I can`t push stream. Restarting OBS does not help, only restarting the computer helps.

### Steps to Reproduce

1. Start pushing RTMP stream
2. Periodically start/stop recording for some time, it usually takes me 8-9 days to reproduce the bug

### Anything else we should know?

OBS studio log url is shortened because of it`s to big. You can search through with "01:00:28.031" - it`s bug reproduce time

**Comment by RytoEX** [comment]
I don't think this is specific to obs-websocket and suspect that this would occur if you directly clicked "Stop Recording" in the UI at this time. I think the best thing that would help is to grab a memory dump of OBS while OBS is frozen. You can [follow the steps here](https://obsproject.com/forum/resources/598/), and then find either myself (RytoEX) or Fenrir on our Discord to send the link to the file. Do not post a link to the memory dump publicly as it could contain sensitive information pertaining to OBS, such as your stream key.

**Comment by mbakumenkov** [comment]
> I don't think this is specific to obs-websocket and suspect that this would occur if you directly clicked "Stop Recording" in the UI at this time. I think the best thing that would help is to grab a memory dump of OBS while OBS is frozen. You can [follow the steps here](https://obsproject.com/forum/resources/598/), and then find either myself (RytoEX) or Fenrir on our Discord to send the link to the file. Do not post a link to the memory dump publicly as it could contain sensitive information pertaining to OBS, such as your stream key.

Thank you so much for ur reply! I will get dump and write you back soon!
